app/views/dradis/plugins/csv/upload/new.html.erb

```diff
@@ -65,3 +68,17 @@
     </div>
   <% end %>
 </div>
+
+<div class="col-12 p-0 order-3 order-xxl-4">
+  <div class="content-container mt-0">
+    <h4 class="header-underline">Output console</h4>
+    <div id="status"></div>
+    <%= content_tag :div, id: 'result', style: 'display:none', data: { url: main_app.status_console_index_path }  do %>
+      <div id="console" class="mx-0 mb-0"></div>
+    <% end %>
+  </div>
+</div>
+
+<script type="text/javascript" charset="utf-8">
+  ConsoleUpdater.jobId = <%= @last_job %>;
+</script>
```

Comment:
I know it's weird, but I got it from: https://github.com/dradis/dradis-ce/blob/develop/app/views/upload/index.html.erb#L81-L83